dev/com.ibm.ws.context/src/com/ibm/ws/context/service/serializable/ThreadContextDescriptorImpl.java

```diff
@@ -147,7 +147,7 @@ public ThreadContextDescriptorImpl(Map<String, String> execProps, @Sensitive byt
                     for (int i = 0; i < contextBytes.length; i++) {
                         String providerName = providerNames.get(i);
                         ThreadContextProvider contextProvider = threadContextMgr.threadContextProviders.getService(providerName);
-                        if (contextProvider == null)
+                        if (contextProvider == null && !"com.ibm.ws.concurrent.mp.cleared.context.provider".equals(providerName))
```

Comment:
What do you think about adding a debug trace if the "com.ibm.ws.concurrent.mp.cleared.context.provider" provider is ignored?

Reply:
I would be willing to add it, but trace for it is already there a couple of lines later:
```
                        if (trace && tc.isDebugEnabled())
                            Tr.debug(this, tc, providerName, contextProvider);
```
This will print the provider name, "com.ibm.ws.concurrent.mp.cleared.context.provider" along with the value null, where for every other provider name, it will print the provider.